components/MobileComponents/MobileSummaryContainer.tsx

```diff
@@ -37,12 +36,31 @@ import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
 import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
 import Grid from "@mui/material/Unstable_Grid2";
 import { AuxData } from "../../src/dataProcessing";
+import getConfig from "next/config";
 
-const dayjs = require('dayjs');
-const relativeTime = require('dayjs/plugin/relativeTime');
+const dayjs = require("dayjs");
+const relativeTime = require("dayjs/plugin/relativeTime");
 dayjs.extend(relativeTime);
 
 let refreshing = false;
+const { publicRuntimeConfig: config } = getConfig();
+
+function getOverallEvaluation(counts: {
```

Comment:
@Akshith-Patil can you export this function from the other file and reuse it instead?

Reply:
yea ill do that